standard_schema/prerelease/HED8.3.0.mediawiki

```diff
@@ -186,13 +186,19 @@ Each term in this vocabulary has a human-readable description and may include ad
 *** Body <nowiki>[The biological structure representing an organism.]</nowiki>
 *** Body-part <nowiki>[Any part of an organism.]</nowiki>
 **** Head <nowiki>[The upper part of the human body, or the front or upper part of the body of an animal, typically separated from the rest of the body by a neck, and containing the brain, mouth, and sense organs.]</nowiki>
+***** Brain <nowiki>[Organ inside the head that is made up of nerve cells and controls the body.]</nowiki>
+****** Frontal-cortex <nowiki>[Frontal part of the cerebral cortex.]</nowiki>
+****** Temporal-cortex <nowiki>[Temporal part of the cerebral cortex.]</nowiki>
+****** Parietal-cortex <nowiki>[Parietal part of the cerebral cortex.]</nowiki>
+****** Occipital-cortex <nowiki>[Occipital part of the cerebral cortex.]</nowiki>

```

Comment:
The four items under Brain have descriptions that are self-defining. Could we get more precise definitions?